### ISSUE TITLE
Thank you box font too small

### ISSUE THREAD

**Issue by sastels** [cds-snc/report-a-cybercrime#1272]
## Summary

Font size in thank you box is too small (16px). Should be 18px.

## Steps to reproduce

go to thankyouPage
inspect the "We've sent you an email with a summary of your report" text
font size is 16px.
